STANDARD.md

```diff
@@ -424,11 +424,29 @@ meshes):
     - example: `(0.0, 100.0, 0.0)` or `(0.5, 0.5, 0.5)`
 
   - `gridUnitSI`
-    - type: *(float64 / REAL8)*
+    - type: 1-dimensional array containing N *(float64 / REAL8)*
+            elements, where N is the number of dimensions in the simulation
```

Comment:
This is related to #125 & #129: if we flatten the array we have to specify *how*.
I would propose to use the wording from #129 as such:
`, dimensions shall be ordered from fastest to slowest varying index [of the described mesh].`

Reply:
update: we probably will go from `slowest to fastest` in #194